public/app/features/explore/ResponseErrorContainer.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { ExploreId, StoreState } from '../../types';
+import { LoadingState } from '@grafana/data';
+import { ErrorContainer } from './ErrorContainer';
+
+interface Props {
+  exploreId: ExploreId;
+}
+export function ResponseErrorContainer(props: Props) {
+  const queryResponse = useSelector((state: StoreState) => state.explore[props.exploreId]?.queryResponse);
+
+  // Only show error if it does not have refId. Otherwise let query row to handle it.
+  const queryError =
+    queryResponse?.state === LoadingState.Error && queryResponse?.error && !queryResponse.error.refId
```

Comment:
Before there was a function that checked for refId recursively anywhere in the error object. Problem is the new backend service also adds config property which on some level can contain refId. This though wasn't used in query row where these errors would not be picked up.